CMakeLists.txt

```diff
@@ -39,9 +39,6 @@ target_sources(${PROJECT_NAME} PRIVATE
 # Handle dependencies
 ###############################################################################
 
-# TODO: This should be set by the user of this library
```

Comment:
Will probably need to update EVT-core to the newest version and include the compiler cmake file which adds the option for the user to select the board to build for